l10n_it_esigibilita_iva/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2017 Alessandro Camilli - Openforce
+# Copyright 2018 Lorenzo Battistini <https://github.com/eLBati>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "ITA - Esigibilita' IVA",
```

Comment:
```suggestion
    "name": "ITA - Esigibilità IVA",
```

Reply:
Vedo che questa modifica è stata fatta, puoi aggiornare la review?

Reply:
> Vedo che questa modifica è stata fatta, puoi aggiornare la review?

Questa l'ho messa come risolta, però era rimasto in piedi questo argomento https://github.com/OCA/l10n-italy/pull/1938#discussion_r531601847